<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-VAQKFJ2" failures="0" tests="4" name="qa.tests.TC_01_VerifyUserAbleToLoginIntoApplication" time="30.189" errors="0" timestamp="15 Apr 2024 09:48:16 GMT" skipped="0">
  <testcase classname="qa.tests.TC_01_VerifyUserAbleToLoginIntoApplication" name="verifyWhetherApplicationProvidAlertMessageWithWrongUserCredentials" time="10.725"/>
  <testcase classname="qa.tests.TC_01_VerifyUserAbleToLoginIntoApplication" name="verifyUserAbleToLoginIntoApplication" time="6.999"/>
  <testcase classname="qa.tests.TC_01_VerifyUserAbleToLoginIntoApplication" name="verifyUserAbleToLoginIntoApplication" time="6.246"/>
  <testcase classname="qa.tests.TC_01_VerifyUserAbleToLoginIntoApplication" name="verifyWhetherApplicationProvidAlertMessageWithWrongUserCredentials" time="6.219"/>
</testsuite> <!-- qa.tests.TC_01_VerifyUserAbleToLoginIntoApplication -->
